.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- HTML Comments can be left in place or removed, dealers choice. They are present simply to guide you on your pull-request journey. --> 
+Proposed changes
+======
+<!-- 
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+-->
+
+Types of changes
+======
+<!-- 
+
+What types of changes does your code introduce to Rancher?
+- Bugfix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+-->
+
+Verification
+======
+<!---
+
+How can the changes be verified? Please provide whatever additional information necessary to help verify the proposed changes.
+
+-->
+
+Linked Issues
+======
+<!--
+
+Link any related issues, pull-requests, or commit hashes that are relavent to this pull-request.
+
+If you are opening a PR without a corresponding issue create an issue before you do, [here](https://github.com/rancher/k3s/issues). This will help QA massively. PR's opened without linked issues will not be merged until an issue is created and linked here. 
```

Comment:
```suggestion
If you are opening a PR without a corresponding issue please consider creating one first, [here](https://github.com/rancher/k3s/issues). This will help QA massively. PR's opened without linked issues will not be merged until an issue is created and linked here. 
```